### DISCUSSION TITLE
Oh no. Error running app. If this keeps happening, please contact support.

### DISCUSSION THREAD

**Discussion by abhirajadhikary06** [code-a2z/jarvis#186] [Q&A] [answered]
Resolve the issue please.

**Comment by Avdhesh-Varshney** [comment]
@abhirajadhikary06 What's the issue?

**Reply by abhirajadhikary06** [reply]
The site hosted on streamlit was not opening

**Reply by Avdhesh-Varshney** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/f4359899-4034-4dcb-a894-58bd57b38231)
It's working perfectly?

**Reply by abhirajadhikary06** [reply]
Yes now it is working
